circle.yml

```diff
@@ -6,15 +6,23 @@ general:
       - production
 machine:
   post:
-    - npm install jscs@1.13.1 -g
-    - npm install mocha@2.4.5 -g
+    - npm install jscs@1.13.1 mocha@2.4.5 -g
+    - cd api && npm run test:e2e website install
+    - cd api && npm run test:e2e app install
```

Comment:
you may want to add

    machine:
      environment:
        NODE_ENV: test
      node:
        version: 5.1.0